### PR TITLE
npm example build step

### PR DESCRIPTION
**Pull request by ericelliott**
Here's the build step for the npm example, complete with browserify + sourcemap support.

### CODE REVIEW

**Comment by ericelliott** [comment]
Note, this depends on the [base implementation PR](https://github.com/ericelliott/todotasks/pull/3). Please merge that, first.

File: implementations/npm/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "name": "todotasks",
+  "version": "1.0.0",
+  "description": "Base project for new taskrunner configurations.",
+  "main": "index.js",
+  "scripts": {
+    "postinstall": "bower install",
+    "test": "node test/test.js",
+    "devbuild": "",
+    "dev": "",
+    "clean": "rm -rf dist/*",
```
Comment (Flet) [inline implementations/npm/package.json]: Should we strive for windows compatibility? `rm` should be changed to `rimraf` module if so

Comment (ericelliott) [inline implementations/npm/package.json]: Good point.

**Comment by ericelliott** [comment]
Fixed.

**Comment by ericelliott** [comment]
Please merge https://github.com/ericelliott/todotasks/pull/3 first.

**Pull request merged**